### ISSUE TITLE
[Feature request] Allow download

### ISSUE THREAD

**Issue by darkdragon-001** [cbenning/fussel#57]
Add button to download original, allow downloading whole album as zip.

**Comment by cbenning** [comment]
Yeah, I think this would be good as a configurable option for sure. I know my use-case I don't want to make it THAT easy but I can see the desire for some cases.
